src/libm-benchmarks/plot-scripts/benchsvml.c

```diff
@@ -0,0 +1,153 @@
+//          Copyright Naoki Shibata 2010 - 2017.
```

Comment:
This looks like a benchmark, why is it in the plot folder? The plot-scripts folder should just contain the scripts that are used to make the plots.